problems/getCountriesSortedByPopulation.js

```diff
@@ -8,7 +8,10 @@
 */
 
 function getCountriesSortedByPopulation() {
-
-}
-
+    let arr = [];
+    return arr.sort(function(country, population) {
+        return population - country 
+    })
+    return []
```

Comment:
This code is unreachable because we have return on line 12 that ends the function.